src/FilteredSearch.tsx

```diff
@@ -7,7 +7,8 @@ const FilteredSearch = styled.div<SxProp>`
   display: flex;
   align-items: stretch;
 
-  summary {
+  summary,
+  > button {
```

Comment:
Would we need to also guard against someone using `as` prop here to change the underlying tag to an anchor or something else?

Reply:
I don't think there would ever be an anchor or something else rendered here